game.py

```diff
@@ -72,9 +68,17 @@ def setup_game(self, next_colors):
         SCREEN.fill(BLACK)
         self.draw_future_grid(next_colors)
         self.draw_grid(True)
-        self.load_score()
-        # self.colored_grid()
-        # self.load_space()
+
+    def handle_spawning(self, next_colors):
+        future_sqr_cord_color = self.draw_predicted(next_colors)
+        self.future_sqr_cord_color = future_sqr_cord_color
+
+    def process_turn(self, next_colors):
+        if self.spawn:
```

Comment:
so if the flag is false we process the turn by doing nothing? That sound sound right.